fe/fe-core/src/main/java/org/apache/doris/datasource/hive/event/AlterTableEvent.java

```diff
@@ -64,6 +65,7 @@ private AlterTableEvent(NotificationEvent event, String catalogName) {
                     (JSONAlterTableMessage) MetastoreEventsProcessor.getMessageDeserializer(event.getMessageFormat())
                             .getAlterTableMessage(event.getMessage());
             tableAfter = Preconditions.checkNotNull(alterTableMessage.getTableObjAfter());
+            tableAfter.setTableName(tableAfter.getTableName().toLowerCase(Locale.ROOT));
```

Comment:
Better not change the `tableTame` field of `tableAfter`, just use a string object such as `afterTableName` to save the lower case table name.